electron/src/webview/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
The entry point of the web content that runs inside Chromium. Just spins up a webview, which gets navigated to our local server that is running `pxt serve`.